src/CommunityToolkit.Maui/Extensions/PopupExtensions.cs

```diff
@@ -0,0 +1,103 @@
+﻿using CommunityToolkit.Maui.Core;
+
+namespace CommunityToolkit.Maui.Views;
+
+/// <summary>
+/// Popup extensions.
+/// </summary>
+public static class PopupExtensions
+{
+	/// <summary>
+	/// Shows a popup with the specified options.
+	/// </summary>
+	/// <param name="page">Current page</param>
+	/// <param name="view">Popup content</param>
+	/// <param name="options"><see cref="PopupOptions"/></param>
+	/// <returns><see cref="PopupResult"/></returns>
+	/// <remarks>This is an <see keyword="async"/> <see keyword="void"/> method. Use <see cref="ShowPopupAsync(Microsoft.Maui.Controls.Page,Microsoft.Maui.Controls.View,CommunityToolkit.Maui.Core.PopupOptions?,System.Threading.CancellationToken)"/> to <see keyword="await"/> this method and return <see cref="PopupResult{T}"/> </remarks>
+	public static void ShowPopup(this Page page, View view, PopupOptions? options = null)
+	{
+		ShowPopup(page.Navigation, view, options);
+	}
+
+	/// <summary>
+	/// Shows a popup with the specified options.
+	/// </summary>
+	/// <param name="navigation">Popup parent</param>
+	/// <param name="view">Popup content</param>
+	/// <param name="options"><see cref="PopupOptions"/></param>
+	/// <returns><see cref="PopupResult"/></returns>
+	/// <remarks>This is an <see keyword="async"/> <see keyword="void"/> method. Use <see cref="ShowPopupAsync(Microsoft.Maui.Controls.Page,Microsoft.Maui.Controls.View,CommunityToolkit.Maui.Core.PopupOptions?,System.Threading.CancellationToken)"/> to <see keyword="await"/> this method</remarks>
+	public static async void ShowPopup(this INavigation navigation, View view, PopupOptions? options)
+	{
+		var popupContainer = new PopupContainer(view, options ?? new(), null);
+
+		await navigation.PushModalAsync(popupContainer, false);
```

Comment:
do we need the `await` here? Since it's `async void` the runtime will just move forward this await. If it's needed, I think makes sense to wrap into a `try/catch` and add `ConfigureAwait(false)`

Reply:
Yup, we always want to `await` every `Task`, including this one.

It's totally fine to not wrap this in a try/catch block. If `Navigation.PushModalAsync()` throws an Exception, it'll be because of an underlying bug in the .NET MAUI framework

We don't own `Task Navigation.PushModalAsync()`, so any exception thrown will point to the .NET MAUI stack.